packages/app/src/app/pages/Sandbox/Editor/Workspace/Dependencies/index.js

```diff
@@ -13,7 +13,7 @@ import ExternalResource from './ExternalResource';
 
 import { ErrorMessage } from './elements';
 
-function Dependencies({ signals, store }) {
+function Dependencies({ signals = null, store = null }) {
```

Comment:
@SaraVieira You can just convert this to a const and use the `useSignals` and `useStore` hooks, no?  🤔

L17 & L23 will crash anyways when `store` would be `null` btw.

```suggestion
const Dependencies = () => {
  {} = useSignals();
  { editor: { currentSandbox: sandbox, parsedConfigurations: { package: { error, parsed }, }, }, } = useStore();
```

Reply:
You are right! All updated